services/controllerhost/controllerhost.go

```diff
@@ -1251,3 +1279,97 @@ func (mcp *Mcp) CreateRemoteZoneConsumerGroupExtent(ctx thrift.Context, createRe
 	lclLg.Info("CreateRemoteZoneConsumerGroupExtent: Extent added to consumer group")
 	return nil
 }
+
+// ValidateDestZoneConfigUpdateRequest validates the zone configs for a UpdateDestinationRequest
+func (mcp *Mcp) ValidateDestZoneConfigUpdateRequest(ctx thrift.Context, logger bark.Logger, updateRequest *shared.UpdateDestinationRequest) (bool, error) {
```

Comment:
I presume you want to name this to not have "update" in it's name .. since you will be probably calling the exact same function from the CreateDestination path too.